<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:b="http://www.google.com/2005/gml/b" xmlns:data="http://www.google.com/2005/gml/data" xmlns:expr="http://www.google.com/2005/gml/expr">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <script src="http://code.jquery.com/jquery.min.js" type="text/javascript"></script>
  <link rel='stylesheet' id='fontawesome-css'  href='http://shalek.wpengine.com/wp-content/themes/proxima/css/font-awesome.min.css?ver=1.4.1' type='text/css' media='all' />
  <title>Jianhong Homepage</title>
  <link rel="stylesheet" href="css/style.css" type="text/css"/>
</head>
<body>
  <div class="headerwide">
	<div class="headertop">
		<div>
		<div class="above_top_bar">
		<span><a href="about.html"><i class="fa fa-map-marker"></i>Find Me</a></span>
		<span><a href='mailto:&#111;&#117;&#046;&#106;&#105;&#097;&#110;&#104;&#111;&#110;&#103;&#064;&#103;&#109;&#097;&#105;&#108;&#046;&#099;&#111;&#109;'><i class="fa fa-envelope-o"></i>ou&#46;jianhong<code>&#64;</code>gmail.com</a></span>
		</div>
		</div>
	</div>
	<div class="headerwrap">
		<div class="titlewrapper">
			<h1 class="title">
				<a href="/">The world of QiuBio</a>
			</h1>
		</div>
		<div class="descriptionwrapper">
			<h2 class="description"><a>Computational Biology & Bioinformatics Blog and Tools</a></h2>
		</div>
	</div>
</div>
<div class="navigationwide">
	<div class="navigationwrap">
		<ul id="navigator">
		<li><a href='about.html'>About</a></li>
		<li><a href='http://qiubio.com'>糗世界</a></li>
		</ul>
	</div>
</div>
<div id="wrapper">
	<div class="itemwrap">
		<h2><a href="packages.html#/1">Bioconductor Packages</a></h2>
		<div>
			
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href='http://qiubio.com'>糗世界</a></h2>
		<div>
			
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href='http://qiubio.com/new'>糗世界2站</a></h2>
		<div>
			
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href='Bioc2020workshop/index.html'>motifStack Workshop for Bioc2020</a></h2>
		<div>
			
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href='trackViewerBiocAsia2020Workshop/articles/trackViewer.html'>trackViewer Workshop for BiocAsia2020</a></h2>
		<div>
			
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href='deepToolsDownstream/articles/deepToolsDownstream.html'>deepToolsDownstream</a></h2>
		<div>
			Downstream plots for deepTools.
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href='HMMtBroadPeak/articles/HMMtBroadPeak.html'>HMMtBroadPeak</a></h2>
		<div>
			The HMMtBroadPeak package is designed to call very broad peaks for data such as lamina-associated domains (LADs), nucleolus-associated domains (NADs), or other topologically associating domains.
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href='uShuffleR/articles/uShuffleR.html'>uShuffleR</a></h2>
		<div>
			uShuffle is a useful tool for shuffling biological sequences published by Minghui Jiang et.al. This package is R wrapper for the tool.
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href='HMMtBroadPeak/articles/HMMtBroadPeak.html'>HMMtBroadPeak</a></h2>
		<div>
			The HMMtBroadPeak package is designed to call very broad peaks for data such as lamina-associated domains (LADs), nucleolus-associated domains (NADs), or other topologically associating domains.
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href="https://jianhong.github.io/genomictools/articles/scripts.html">Basic</a> and <a href="https://jianhong.github.io/workshop2020/">Advanced</a> boostcamp</h2>
		<div>
			<p><strong>» Bioinformatics Basics/Advanced boostcamp: Sequence Analysis Bootcamp</strong></p>
			<p>This is a series of lectures on Unix, DCC cluster, R language and bioinformatics tools for molecular biologist. The main goal is to introduce the audience to the basics of bioinformatics with a focus on deep sequencing data analysis using a mixed data of RNA-seq and ChIP-seq as an example. Basic computer skills are required to follow the lectures. No programming skills are required. The boost camp stats with a quick introduction to how to read the analysis results. Next, genomic file formats will be introduced. Then the analysis of RNA-seq and ChIP-seq will be demonstrated. This is followed by the visualizing and downstream analyzing of the data. The last part of this boost camp is a brief introduction to R programming language. </p>
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href="http://bioconductor.org/packages/release/bioc/html/ribosomeProfilingQC.html">ribosomeProfilingQC</a>
			<a href="http://bioconductor.org/packages/release/bioc/html/ribosomeProfilingQC.html"><img src="http://bioconductor.org/shields/availability/3.13/ribosomeProfilingQC.svg"></a>
			<a href="http://bioconductor.org/checkResults/devel/bioc-LATEST/ribosomeProfilingQC/"><img src="http://bioconductor.org/shields/build/devel/bioc/ribosomeProfilingQC.svg"></a>
		</h2>
		<div>
			<p><strong>» Ribosome Profiling Quality Control</strong></p>
			<p>Ribo-Seq (also named ribosome profiling or footprinting) measures translatome (unlike RNA-Seq, which sequences the transcriptome) by direct quantification of the ribosome-protected fragments (RPFs). This package provides the tools for quality assessment of ribosome profiling. In addition, it can preprocess Ribo-Seq data for subsequent differential analysis.</p>
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href="http://bioconductor.org/packages/release/bioc/html/motifStack.html">motifStack</a>
			<a href="https://doi.org/10.1038/nmeth.4555"><img src="https://img.shields.io/badge/DOI-10.1038%2Fnmeth.4555-brightgreen.svg"></a>
			<a href="http://bioconductor.org/packages/devel/bioc/html/motifStack.html"><img src="http://bioconductor.org/shields/availability/devel/motifStack.svg"></a>
			<a href="http://bioconductor.org/checkResults/devel/bioc-LATEST/motifStack/"><img src="http://bioconductor.org/shields/build/devel/bioc/motifStack.svg"></a>
		</h2>
		<div>
			<p><strong>» a bioconductor package to plot stacked logos for single or multiple DNA, RNA and amino acid sequence</strong></p>
			<p>motifStack is a package that is able to draw amino acid sequence as easy as to draw DNA/RNA sequence. motifStack provides the flexibility for users to select the font type and symbol colors. motifStack is designed for graphical representation of multiple motifs.</p>
			<p>This package is involved in the <a href="http://www.bioconductor.org/help/workflows/generegulation/">pipeline of finding candidate binding sites for known transcription factors via sequence matching</a>.</p>
			<p class="paper">Ou J, Wolfe SA, Brodsky MH, Zhu LJ (2018). “motifStack for the analysis of transcription factor binding site evolution.” Nature Methods, 15, 8-9. <a href="http://dx.doi.org/10.1038/nmeth.4555">doi: 10.1038/nmeth.4555</a>.</p>
		</div>
	</div>	
	<div class="itemwrap">
		<h2><a href="http://bioconductor.org/packages/release/bioc/html/ATACseqQC.html">ATACseqQC</a>
			<a href="https://doi.org/10.1186/s12864-018-4559-3"><img src="https://img.shields.io/badge/DOI-10.1186%2Fs12864--018--4559--3-brightgreen.svg"></a>
			<a href="http://bioconductor.org/packages/devel/bioc/html/ATACseqQC.html"><img src="http://bioconductor.org/shields/availability/devel/ATACseqQC.svg"></a>
			<a href="http://bioconductor.org/checkResults/devel/bioc-LATEST/ATACseqQC/"><img src="http://bioconductor.org/shields/build/devel/bioc/ATACseqQC.svg"></a>
		</h2>
		<div>
			<p><strong>» a bioconductor package for ATAC-seq Quality Control</strong></p>
			<p>ATAC-seq, an assay for Transposase-Accessible Chromatin using sequencing, is a rapid and sensitive method for chromatin accessibility analysis. 
			It was developed as an alternative method to MNase-seq, FAIRE-seq and DNAse-seq. 
			Comparing to the other methods, ATAC-seq requires less amount of the biological samples and time to process. 
			In the process of analyzing several ATAC-seq dataset produced in our labs, we learned some of the unique aspects of the quality assessment for ATAC-seq data.
			To help users to quickly assess whether their ATAC-seq experiment is successful, we developed ATACseqQC package partially following the guideline published in Nature Method 2013 (Greenleaf et al.), including diagnostic plot of fragment size distribution, proportion of mitochondria reads, nucleosome positioning pattern, and CTCF or other Transcript Factor footprints.</p>
			<p class="paper">Ou J, Liu H, Yu J, Kelliher MA, Castilla LH, Lawson ND, Zhu LJ (2018). “ATACseqQC: a Bioconductor package for post-alignment quality assessment of ATAC-seq data.” BMC Genomics, 19(1), 169. ISSN 1471-2164, <a href="https://doi.org/10.1186/s12864-018-4559-3">doi: 10.1186/s12864-018-4559-3</a>.</p>
		</div>
	</div>	
	<div class="itemwrap">
		<h2>
			<a href="http://bioconductor.org/packages/release/bioc/html/trackViewer.html">trackViewer</a>
			<a href="https://doi.org/10.1038/s41592-019-0430-y"><img src="https://img.shields.io/badge/DOI-10.1038%2Fs41592--019--0430--y-brightgreen.svg"></a>
			<a href="http://bioconductor.org/packages/devel/bioc/html/trackViewer.html"><img src="http://bioconductor.org/shields/availability/devel/trackViewer.svg"></a>
			<a href="http://bioconductor.org/checkResults/devel/bioc-LATEST/trackViewer/"><img src="http://bioconductor.org/shields/build/devel/bioc/trackViewer.svg"></a>
		</h2>
		<div>
			<p><strong>» A bioconductor package with minimalist design for plotting elegant track layers</strong></p>
			<p>Visualize mapped reads along with annotation as track layers for NGS dataset such as ChIP-seq, RNA-seq, miRNA-seq, DNA-seq.</p>
			<p class="paper">Ou, J., Zhu, L.J. trackViewer: a Bioconductor package for interactive and integrative visualization of multi-omics data. Nat Methods 16, 453–454 (2019). <a href="https://doi.org/10.1038/s41592-019-0430-y">doi: 10.1038/s41592-019-0430-y</a></p>
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href="http://bioconductor.org/packages/release/bioc/html/ChIPpeakAnno.html">ChIPpeakAnno</a>
			<a href="http://bioconductor.org/packages/devel/bioc/html/ChIPpeakAnno.html"><img src="http://bioconductor.org/shields/availability/devel/ChIPpeakAnno.svg"></a>
			<a href="http://bioconductor.org/checkResults/devel/bioc-LATEST/ChIPpeakAnno/"><img src="http://bioconductor.org/shields/build/devel/bioc/ChIPpeakAnno.svg"></a>
		</h2>
		<div>
			<p><strong>» a bioconductor package for annotating peaks identified from ChIP-seq, Chip-chip or any high-throuput experiments</strong></p>
			<p>Batch annotation of the peaks identified from either ChIP-seq or ChIP-chip experiments.
			The package includes functions to retrieve the sequences around the peak, obtain enriched Gene Ontology (GO) terms, find the nearest gene, exon, miRNA or custom features such as most conserved elements and other transcription factor binding sites supplied by users. This package leverages the biomaRt, IRanges, Biostrings, BSgenome, GO.db, multtest and stat packages</p>
			<p class="paper">Zhu LJ*, Gazin C, Lawson ND, Pages H, Lin SM, Lapointe DS and Green MR. (2010) [* denotes corresponding author] ChIPpeakAnno: a Bioconductor package to annotate ChIP-seq and ChIP-chip data. BMC Bioinformatics 2010, 11:237.</p>
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href="http://bioconductor.org/packages/release/bioc/html/GeneNetworkBuilder.html">GeneNetworkBuilder</a>
			<a href="http://bioconductor.org/packages/devel/bioc/html/GeneNetworkBuilder.html"><img src="http://bioconductor.org/shields/availability/devel/GeneNetworkBuilder.svg"></a>
			<a href="http://bioconductor.org/checkResults/devel/bioc-LATEST/GeneNetworkBuilder/"><img src="http://bioconductor.org/shields/build/devel/bioc/GeneNetworkBuilder.svg"></a>
		</h2>
		<div>
			<p><strong>» Build Regulatory Network from ChIP-chip/ChIP-seq and Expression Data</strong></p>
			<p>GeneNetworkBuilder (GNB) is an appliation for discovering the transcriptional regulatory network for a given transcription factor (TF) of Caenorhabditis elegans, Homo sapiens and so on, using ChIP-chip (ChIP-seq) combined with gene expression profile from either RNA-seq or expression microarray experiments.</p>
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href="http://bioconductor.org/packages/release/bioc/html/dagLogo.html">dagLogo</a>
			<a href="https://doi.org/10.1371/journal.pone.0242030"><img src="https://img.shields.io/badge/DOI-10.1371%2Fjournal.pone.0242030-brightgreen.svg"></a>
			<a href="http://bioconductor.org/packages/devel/bioc/html/dagLogo.html"><img src="http://bioconductor.org/shields/availability/devel/dagLogo.svg"></a>
			<a href="http://bioconductor.org/checkResults/devel/bioc-LATEST/dagLogo/"><img src="http://bioconductor.org/shields/build/devel/bioc/dagLogo.svg"></a>
		</h2>
		<div>
			<p><strong>» a bioconductor package to find and visualize signficantly enriched or depleted amino acid motif or amino acid group patterns in proteom dataset</strong></p>
			<p>In addition to implement iceLogo in R to visualize differential amino acid sequence pattern, dagLogo can also test and visualize significant 
amino acid group patterns by classifying the amino acids into groups according to charge, chemistry and hydrophobicity
and etc.</p>
            <p class="paper">Ou, Jianhong, Liu, Haibo, Nirala, K N, Stukalov, Alexey, Acharya, Usha, Green, R M, Zhu, Julie L (2020). “dagLogo: An R/Bioconductor package for identifying and visualizing differential amino acid group usage in proteomics data.” Plos one, 15(11), e0242030. <a href="https://doi.org/10.1371/journal.pone.0242030">>doi: 10.1371/journal.pone.0242030</a>.</p>
		</div>
	</div>
	<div class="itemwrap">
		<h2><a href="http://bioconductor.org/packages/release/bioc/html/InPAS.html">InPAS</a>
			<a href="http://bioconductor.org/packages/devel/bioc/html/InPAS.html"><img src="http://bioconductor.org/shields/availability/devel/InPAS.svg"></a>
			<a href="http://bioconductor.org/checkResults/devel/bioc-LATEST/InPAS/"><img src="http://bioconductor.org/shields/build/devel/bioc/InPAS.svg"></a>
		</h2>
		<div>
			<p><strong>» Identification of Novel alternative PolyAdenylation Sites (PAS)</strong></p>
			<p>Alternative polyadenylation (APA) is one of the important post-transcriptional regulation mechanisms which occurs in most human genes. InPAS facilitates the discovery of novel APA sites from RNAseq data. It leverages cleanUpdTSeq to fine tune identified APA sites.</p>
		</div>
	</div>
</div>
<div class="footerwide">
	<div class="footerwrap">
		<div class="footerleft">© 2017 <a href="#" target="_balnk">jianhong@qiubio</a>
		</div>
		<div class="footerright">
				This site is maintained by <a href='http://qiubio.com:8080/'>Jianhong Ou</a>.<br/>
		</div>
	</div>
</div>
</body>
</html>
